### ISSUE TITLE
About project status

### ISSUE THREAD

**Issue by damooo** [ArchiveLabs/iiif.archivelab.org#49]
Namaste @mekarpeles,

thanks for iiif api.

We want to make many indic texts in archive.org to be transcribed, annotated, and analyzed to make them amenable for further usage, and research at large scale. Both printed and handwritten. We also had written software and UI workflows for this, where we can import any iiif book as source. It will be great if `archive.org` provides first class support to iiif in it's truest sense, as most of eco-system, R&D, UIs are moving towards iiif.

Is there any possibility, we can depend upon iiif.archivelab.org api? Will it come out of beta?.

I have good experience with developing iiif image, presentation apis with high performance `loris` and `iiif_prezi`. If it is possible for `iiif` to be supported officially, i can dedicate my time to re-implement both apis with above established tools, as flask apis.

Thanks again for your time and work.

**Comment by mekarpeles** [comment]
It's been around for 3+ years now. It's not heavily maintained but it's fairly heavily used.

In the worst case scenario, anyone can setup this service **outside** of Internet Archive and it will still work.

They currently are Flask APIs.

**Comment by damooo** [comment]
Yes, great to hear that api is in heavy usage for considerable time. Till now we are banking on worst case scenario only. I will try to contribute wherever i can.

Can we consider `iiif2` package also as being maintained? Or should we integrate `loris` as engine? `loris` also maintains computed-resources at different stages as cache, along with actual resources. It seems is in active development.

**Comment by mekarpeles** [comment]
You're welcome to the code over to the loris engine if you'd like -- I
originally authored iiif2 so I try to update it when there are issues, but
I've had my hands full with running OpenLibrary.org over at the Internet
Archive.

Happy to support you in what ways I can!

Thank you

best wishes,

- mek

https://mek.fyi
Citizen of the World
Open Librarian @ openlibrary.org
